CONTRIBUTING.md

```diff
@@ -130,3 +130,15 @@ To fix the issue, you need to update the Maven importing settings:
 To import Quarkus you need to define the JVM Max Heap Size (E.g. `-Xmx1g`)
 
 **Note*:* As for now, we can't provide a unique Max Heap Size value. We have been reported to require from 768M to more than 3G to import Quarkus properly.
```

Comment:
Should be `**Note**` ?

Reply:
Oops, saw that this was not part of your change. Could you nevertheless fix this as well?

Reply:
yes, of course

Reply:
Thanks!